test/test_quantization.py

```diff
@@ -318,5 +320,55 @@ def checkQuantized(model):
         model = quantize_qat(model, test_only_train_fn, self.img_data)
         checkQuantized(model)
 
+
+class FusionTest(QuantizationTestCase):
+    def test_fuse_module_train(self):
+        import torch.nn._intrinsic.modules.fused as torch_fused
+        testMod = ModForFusion()
+        testMod.train()
+        fuse_modules(testMod, [['conv1', 'bn1', 'relu1'],
+                               ['sub1.conv', 'sub1.bn']])
+        self.assertEqual(type(testMod.conv1), torch_fused.ConvBnReLU2d,
+                         "Fused Conv + BN + Relu first layer")
+        self.assertEqual(type(testMod.bn1), torch.nn.Identity,
+                         "Fused Conv + BN + Relu (skipped BN)")
+        self.assertEqual(type(testMod.relu1), torch.nn.Identity,
+                         "Fused Conv + BN + Relu (skipped Relu)")
+
+        self.assertEqual(type(testMod.sub1.conv), torch_fused.ConvBn2d,
+                         "Fused submodule Conv + BN")
+        self.assertEqual(type(testMod.sub1.bn), torch.nn.Identity,
+                         "Fused submodule Conv + BN (skipped BN)")
+        self.assertEqual(type(testMod.sub2.conv), torch.nn.Conv2d,
+                         "Non-fused submodule Conv")
+        self.assertEqual(type(testMod.sub2.bn), torch.nn.BatchNorm2d,
+                         "Non-fused submodule BN")
+
+    def test_fuse_module_eval(self):
```

Comment:
do you want to add a test for numerics? specifically Conv+BN?

Reply:
Please add tests for numerics for both eval and training cases for conv+bn, conv+bn+relu.

Reply:
I think testing for conv+bn is sufficient because +relu version of it reuses?